Gopkg.toml

```diff
@@ -107,8 +107,9 @@ required = [
 [[constraint]]
   branch = "master"
   name = "github.com/metal3-io/cluster-api-provider-baremetal"
-  source = "github.com/openshift/cluster-api-provider-baremetal/pkg/apis"
+  source = "https://github.com/openshift/cluster-api-provider-baremetal/pkg/apis"
```

Comment:
looks unrelated?

Reply:
looks like CAPBM got updated, but not BMO

Reply:
There's no update to BMO, I'm only pulling in the BMC package which hasn't had changes.

Reply:
Shouldn't we enforce it to be https?

Reply:
ok, that's fine, the PR title and commit msg just referred to BMO, not this component.

Reply:
I've fixed the commit